### PR TITLE
Jenkins pipeline scripts allowing parallel MTR execution.

### PR DESCRIPTION
**Pull request by kamil-holubicki**
All branches/repos point right not to my fork. Will be adjusted after approval, before merge.

https://ps80.cd.percona.com/view/Parallel%20MTR/

### CODE REVIEW

File: jenkins/param-parallel-mtr.yml [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+        - "yes"
+        - "no"
+        description: Run mtr --suite=keyring_vault
+    - string:
+        name: KEYRING_VAULT_V1_VERSION
+        default: "0.9.6"
+        description: Specifies version of Hashicorp Vault for V1 tests
+    - string:
+        name: KEYRING_VAULT_V2_VERSION
+        default: "1.9.0"
```
Comment (percona-ysorokin) [inline jenkins/param-parallel-mtr.yml:124]: We can update this to a newer version `1.13.1` (Apr 13, 2023 as per https://developer.hashicorp.com/vault/downloads)

File: jenkins/pipeline-parallel-mtr.groovy [hunk 2]
```diff
@@ -0,0 +1,833 @@
+PIPELINE_TIMEOUT = 24
+JENKINS_SCRIPTS_BRANCH = 'parallel-mtr-refactor'
+JENKINS_SCRIPTS_REPO = 'https://github.com/kamil-holubicki/ps-build'
+//JENKINS_SCRIPTS_BRANCH = '8.0'
```
Comment (percona-ysorokin) [inline jenkins/pipeline-parallel-mtr.groovy]: Just don't forget to change this to the main repo

File: jenkins/pipeline-parallel-mtr.yml [hunk 3]
```diff
@@ -0,0 +1,197 @@
+- job:
+    name: percona-server-8.0-pipeline-parallel-mtr-refactor
+    project-type: pipeline
+    description: |
+        Do not edit this job through the web!
+    concurrent: true
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/kamil-holubicki/ps-build
```
Comment (percona-ysorokin) [inline jenkins/pipeline-parallel-mtr.yml]: Don't forget to change this as well

**Review by percona-ysorokin — APPROVED** [review]
LGTM

File: jenkins/param-parallel-mtr.yml (hunk 1 above)
Comment (kamil-holubicki) [reply jenkins/param-parallel-mtr.yml:124]: Let's do it afterward

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**